cypress_shared/components/bedspaceSearchResult.ts

```diff
@@ -9,12 +9,12 @@ export default class BedspaceSearchResult extends Component {
   shouldShowResult(checkCount = true): void {
     cy.get('div[data-cy-result]')
```

Comment:
Do we want/need this any more? Or should we just find the table?

Reply:
I thought about this. We could just match on `table` though I figured this was a convention to prevent fragile matchers should there be multiple tables on the same page? I'll remove it and we can cross with that bridge later if we come to it.